jsapp/scss/components/_kobo.form-view.scss

```diff
@@ -136,6 +136,12 @@
       vertical-align: middle;
     }
   }
+
+  label.form-view__tab-label {
```

Comment:
I think this got removed with some cleanup (or by some merge going bad?), so you'd need to bring it back in the places that render `form-view__tab` (previously they used `bem.FormView__tab`, but it got simplified). 

But instead of going with `form-view__tab-label` and `width: 70%`, try setting `display: flex` on `.form-view__tab` (and in `_kobo.form-view.scss` where it belongs instead of here)

PS. could you delete `bem.FormView__tab` from `bem.es6`?